docs/source/cheat_sheet.rst

```diff
@@ -11,121 +11,158 @@ language represents various common types in Python 2.
    many of the examples have a dual purpose: show how to write the
    annotation, and show the inferred types.
 
+
+Built-in types
+**************
+
 .. code-block:: python
 
-   from typing import List, Dict, Set, Tuple, Union, Optional, Callable, Match
-   from six import text_type
+   from typing import List, Set, Dict, Tuple, Optional
+   import six
 
-   # For builtin types, just use the name of the type
+   # For simple built-in types, just use the name of the type.
    x = 1 # type: int
    x = 1.0 # type: float
    x = "test" # type: str
    x = u"test" # type: unicode
 
-   # Print an error message giving the type of x (it's also a runtime
-   # error, so don't leave it in)
-   reveal_type(x)
-
    # For collections, the name of the type is capitalized, and the
-   # name of the type in the collection is in brackets.
+   # name of the type inside the collection is in brackets.
    x = [1] # type: List[int]
    x = set([6, 7]) # type: Set[int]
-   # For mappings, we need the types of both keys and values
+
+   # For mappings, we need the types of both keys and values.
    x = dict(field=2.0) # type: Dict[str, float]
-   # For tuples, we specify the types of all the elements
+
+   # For tuples, we specify the types of all the elements.
    x = (3, "yes", 7.5) # type: Tuple[int, str, float]
 
-   # six.text_type is str/unicode in Python 2 and string but not bytes in Python 3
-   x = ["string", u"unicode"] # type: List[text_type]
-   # If something could be one of a few types, use Union
-   x = [3, 5, "test", "fun"] # type: List[Union[int, str]]
-   x = re.match(r'[0-9]+', "15") # type: Match[str]
+   # For textual data, we generally use six.text_type.
```

Comment:
How should we introduce AnyStr? It's important but would be confusing this early.